.github/workflows/ci-actions-incremental.yml

```diff
@@ -389,6 +409,14 @@ jobs:
           path: .
       - name: Extract .m2/repository/io/quarkus
         run: tar -xzf m2-io-quarkus.tgz -C ~
```

Comment:
Interestingly, on top of all the GitHub Actions caching, you could store the Develocity local cache from the initial build here, instead of just the artifacts... Which would potentially speed up the first build of every PR. You'd skip compilation of integration tests, for instance.

However, this would probably only be beneficial if the caching below overrides local caches on the second build.